src/prototypes/single-page/example/example.scss

```diff
@@ -1,9 +1,46 @@
 @use "../../../mixins/ms";
 @use "../../../compiled/tokens/scss/size";
+@use "../../../compiled/tokens/scss/breakpoint";
+
 
 #single-page {
   .main-container {
     padding-top: ms.step(6);
     padding-bottom: ms.step(6);
+    position: relative;
+  }
+
+  .alert-container {
+    position: absolute;
+    background-color: white;
+    box-shadow: 0 0 0 2px hsla(224,65%,16%,0.2),
+    4px 4px 0 2px hsla(224,65%,16%,0.1);
+    top: -.5em;
+    left: 10%;
+    @media (min-width: breakpoint.$l) {
+      top: -1.5em;
+      left: 35%;
+    }
+
+    .icon {
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: 50px;
+      height: 100%;
+      background-color: #ffdebd;
+      border-top-left-radius: size.$border-radius-medium;
+      border-bottom-left-radius: size.$border-radius-medium;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      @media (min-width: breakpoint.$l) {
+        width: 60px;
+      }
+    }
+
+    .text {
+      margin-left: 3em;
+    }
```

Comment:
I would not expect these styles to still be necessary in this prototype if you're using the component.